lib/convenient_service/support/dependency_container/import.rb

```diff
@@ -9,12 +9,13 @@ module Import
         class_methods do
           ##
           # @param full_name [String, Symbol]
+          # @param as [String, Symbol, nil]
           # @param from [Module]
           # @param scope [:instance, :class]
           # @param prepend [Boolean]
           # @return [ConvenientService::Support::DependencyContainer::Entities::Method]
           #
-          def import(full_name, from:, scope: Constants::DEFAULT_SCOPE, prepend: Constants::DEFAULT_PREPEND)
+          def import(full_name, from:, as: nil, scope: Constants::DEFAULT_SCOPE, prepend: Constants::DEFAULT_PREPEND)
```

Comment:
**Critical**

Please, utilize `Support::NOT_PASSED` when `as` is not passed.

Thanks.